src/app/src/components/FacilitiesMap.jsx

```diff
@@ -226,6 +227,13 @@ function FacilitiesMap({
         setFacilitiesToDisambiguate,
     ]);
 
+    useEffect(() => {
+        // Close multiple facilities popup on fresh searches
+        if (fetching) {
+            setFacilitiesToDisambiguate(null);
+        }
+    }, [fetching]);
```

Comment:
This is probably fine but it's interesting that the linter didn't complain about `setFacilitiesToDisambiguate` not being included in the dependencies array. In practice that function should never change during the component lifecycle, so maybe the linting rule is robust enough to know that too.

Reply:
Reading up here: https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies, I see:

> The identity of the `setCount` function is guaranteed to be stable so it’s safe to omit.

Where `setCount` was from `useState()`:

```js
const [count, setCount] = useState();
```

So I think we should be good.